Classes/EventPublisher/DeferEventPublisher.php

```diff
@@ -0,0 +1,72 @@
+<?php
+declare(strict_types=1);
+namespace Neos\EventSourcing\EventPublisher;
+
+/*
+ * This file is part of the Neos.EventSourcing package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\EventSourcing\Event\DomainEvents;
+
+/**
+ * An Event Publisher that doesn't publish the events immediately but stores them until the end of the current request
```

Comment:
. This depends on Flow's shutdownObject() methods being called at the end of the request cycle.

Reply:
Yes, or rather on `invoke()`. Do you suggest to make that clearer in the doc comment of the class?

Reply:
yes :)